LayoutTests/fast/harness/results.html

```diff
@@ -1,5 +1,27 @@
 <!DOCTYPE html>
+<html>
+<head>
 <style>
+:root {
+    color-scheme: dark light;
+    --clickable-link-color: blue;
+    --item-highlight-color: red;
+    --button-background-color: white;
+    --panel-background-color: rgba(255, 255, 255, 0.9);
+}
+
+@media (prefers-color-scheme: dark) {
+    :root {
+        --item-highlight-color: #F8490A;
+        --button-background-color: #666;
+        --panel-background-color: rgba(50, 50, 50, 0.9);
+        --clickable-link-color: #9e9eff;
+    }
+    a:link { color: #9e9eff; }
+    a:visited { color: #d0adf0; }
+    a:active { color: red; }
```

Comment:
I assume these are due to https://bugs.webkit.org/show_bug.cgi?id=209851? (which I really need to look into...)

If so, can we leave a comment with that bug number, so that we can try to remove this CSS in the future?

Reply:
Yes, they are.